### PR TITLE
finalize english corrections

### PR DESCRIPTION
**Pull request by lemelon**

### CODE REVIEW

**Comment by clochix** [comment]
Thanks @lemelon ☺
To whoever will merge this, please don't forget to push the source file to Transifex: `tx push --source`. Thanks!

File: src/locales/en.json [hunk 1]
```diff
@@ -99,11 +99,11 @@
 	"architecture introduction": "Peek under Cozy's hood, and learn what makes it tick.",
 	"architecture introduction two": "Behind Cozy's polished and user-friendly interface hides an advanced platform based on modern web technologies. To get a general idea of Cozy's overall architecture, you can think of it as a cake with two layers: Apps and Data.",
 	"architecture apps layer": "The main components in the Apps layer include the Controller service responsible for deploying, removing, and updating apps. Controller has its own web-based graphical interface called the Home that lets you administer Cozy: deploy, update, and remove applications as well as modify settings like language, timezone, etc.",
-	"architecture datasystem": "The Apps layer sits behind a proxy that acts as a dispatcher, redirecting requests received from external clients (browsers, native apps, third-party services, etc.) to the appropriate apps. The proxy also handles authentication and authorization in Cozy, including registration, login, logout, and password reset operations.",
```
Comment (aenario) [inline src/locales/en.json]: mmmmh, looks like architecture datasystem & architecture proxy are reversed. Is it normal ?

File: src/locales/en.json [hunk 2]
```diff
@@ -115,33 +115,33 @@
 	"community dev title": "Hack & Develop",
 	"community mentorship button": "Apply to the Mentorship Program",
 	"community hack button": "Read our tutorial",
-	"community dev content": "Speaking of developers, there is a lot you can do with Cozy as a coder. You can create a connector or you can build a complete Cozy app, and our documentation can help you to get started. If you need a helping hand with your Cozy hacking and coding endeavors, you are welcome to apply for our mentorship program.",
+	"community dev content": "For developers, there is a lot there is a lot you can do with Cozy. You can create a connector or you can build a complete Cozy app, and our documentation can help you to get started. If you need a helping hand with your Cozy hacking and coding endeavors, you are welcome to apply for our mentorship program.",
```
Comment (aenario) [inline src/locales/en.json]: You doubled there is a lot

**Comment by frankrousseau** [comment]
@aenario could you finish this review?

**Pull request merged**